pkg/libinit/root_linux.go

```diff
@@ -214,6 +225,10 @@ func CreateRootfs() {
 	if !present || boolErr != nil || !systemdEnabled {
 		create(cgroupsnamespace, true)
 	}
+
+	// Prepare default etc files under tmpfs
+	ioutil.WriteFile("/tmp/resolv.conf", []byte(nameserver), 0644)
```

Comment:
This won't work for those building off an existing initramfs. See this code path https://github.com/u-root/u-root/blob/master/u-root.go#L240 which does not call the DefaultRamfs() function.

Reply:
You are right. However, if a user specify a initramfs, we should honor their etc files. No need to replace the etc files with uroot's default. Thus, this `/tmp/resolv.conf` will not be effective. It's by design. Same reason as the comments above. This uroot's default resolv.conf should only be used in uroot's default initrd.